src/plugins/unified_field_list/public/components/field_stats/field_stats.tsx

```diff
@@ -240,39 +234,86 @@ const FieldStatsComponent: React.FC<FieldStatsProps> = ({
   const formatter = dataView.getFormatterForField(field);
   let title = <></>;
 
-  if (field.type.includes('range')) {
-    return (
-      <>
-        <EuiText size="s">
-          {i18n.translate('unifiedFieldList.fieldStats.notAvailableForRangeFieldDescription', {
-            defaultMessage: `Summary information is not available for range type fields.`,
-          })}
-        </EuiText>
-      </>
+  function combineWithTitleAndFooter(el: React.ReactElement) {
+    const countsElement = totalDocuments ? (
+      <EuiText color="subdued" size="xs" data-test-subj={`${dataTestSubject}-statsFooter`}>
+        {sampledDocuments && sampledDocuments < totalDocuments ? (
+          <FormattedMessage
+            id="unifiedFieldList.fieldStats.calculatedFromSampleRecordsLabel"
+            defaultMessage="Calculated from sample of {sampledDocumentsFormatted} {sampledDocuments, plural, one {record} other {records}}"
```

Comment:
@gchaps New label "Calculated from sample of X records"

Reply:
Calculated from xxx sample records.

Include ending period.